ant-evm/src/data_payments.rs

```diff
@@ -50,30 +50,36 @@ impl ProofOfPayment {
         self.peer_quotes
             .clone()
             .into_iter()
-            .map(|(_, quote)| (quote.hash(), quote.quoting_metrics, quote.rewards_address))
+            .map(|(_, _, quote)| (quote.hash(), quote.quoting_metrics, quote.rewards_address))
             .collect()
     }
 
     /// returns the list of payees
-    pub fn payees(&self) -> Vec<PeerId> {
+    pub fn payees(&self) -> Vec<(PeerId, Vec<Multiaddr>)> {
```

Comment:
This is a massive change and will be breaking many things, just want to understand why we are doing this, could you mind explaining why we are doing this?